http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoPubsubMessageSubscriber.scala

```diff
@@ -713,22 +717,21 @@ class LeoPubsubMessageSubscriber[F[_]: Timer: ContextShift: Parallel](
     for {
       ctx <- ev.ask
 
-      // The "create app" flow does a number of things:
```

Comment:
I decided to change the nodepool creation to be asynchronous. The cluster creation is still synchronous to the message. The reason is that nodepool operations may potentially have long delays due to locks, so probably better to be async. Also the locking code is cleaner if `createNodepool` and `pollNodepool` aren't split up as separate methods.